packages/console/src/hooks/use-subscribe.ts

```diff
@@ -0,0 +1,79 @@
+import { nanoid } from 'nanoid';
+import { toast } from 'react-hot-toast';
+
+import { useCloudApi } from '@/cloud/hooks/use-cloud-api';
+import { type CreateTenantData } from '@/components/CreateTenantModal/type';
+import { getBasename } from '@/consts';
+import { checkoutStateQueryKey, checkoutSuccessCallbackPath } from '@/consts/subscriptions';
+import { createLocalCheckoutSession } from '@/utils/checkout';
+
+type SubscribeProps = {
+  planId: string;
+  callbackPage?: string;
+  tenantId?: string;
+  tenantData?: CreateTenantData;
```

Comment:
Doesn't tenantData contain tenantId?

Reply:
The tenant will be created after checkout in the backend, so we have no tenant id at this time.